extensions/notebook/package.json

```diff
@@ -657,5 +657,8 @@
     "typemoq": "^2.1.0",
     "vscodetestcover": "^1.1.0"
   },
+  "resolutions": {
+    "url-parse": "^1.5.0"
```

Comment:
Latest is 1.5.1. Can we just move to that instead?

Reply:
I'll update this - it actually pulled in 1.5.1 already though (the ^ versioning lets it pull in minor updates instead)